app/models/cms/portlet.rb

```diff
@@ -43,11 +43,11 @@ def self.model_name
     end
 
     def self.human_name
-      I18n.t(:name, :scope => [:cms, :portlet, name.underscore])
+      I18n.t(:name, :scope => [:cms, :portlet, name.underscore], default: nil)
```

Comment:
This is similar to builtin legal terms. The error was:
```
Translation missing: en.cms.portlet.cms/portlet/base.name
```

Normally, portlets are initialized as sublclasses, `ExternalRssFeedPortlet`, `LatestForumPostsPortlet`, `TableOfContentsPortlet`, which do have [translations](https://github.com/3scale/porta/blob/rails71/config/locales/en.yml#L1534-L1544).

When the base class is initialized (like in `models_test.rb`), an exception is thrown, because there is no entry for `cms/portlet/base`.

Reply:
While portlets probably have to be removed anyway, I'd suggest
```suggestion
      I18n.t(:name, :scope => [:cms, :portlet, name.underscore], default: name)
```

This is to avoid just getting `nil`. But I don't think this feature is even used so perhaps you can ignore that comment.

Reply:
Right, and also I don't think we care about portlets :upside_down_face: 
We probably should remove this feature from the code base.